doc/installation.rst

```diff
@@ -232,63 +203,117 @@ Optional modules
 Installation procedure
 ~~~~~~~~~~~~~~~~~~~~~~
 
-If you use Anaconda, you need to run this command to install Spyder:
+You can install Spyder with the ``pip`` package manager, which comes by
+default with most Python installations.
+Before installing Spyder itself by this method, you need at least:
 
-   ``conda install spyder``
+* `The Python programming language <http://www.python.org/>`_
+* `PyQt5 <http://www.riverbankcomputing.co.uk/software/pyqt/download5>`_ (recommended)
+  or `PyQt4 <http://www.riverbankcomputing.co.uk/software/pyqt/download>`_
 
-If you don't use Anaconda, you need to run:
 
-   ``pip install --upgrade spyder``
+Then, to install Spyder and its other dependencies, run ``pip install spyder``
+
+**Important note**: This does not install the graphical Python libraries (i.e.
+PyQt5 or PyQt4) that Spyder depends on. Those have to be installed separately
+after installing Python. Please see their documentation for more on that.
```

Comment:
As I said above, this is only true now for Python 2.

Reply:
Yes, fixed already as mentioned above.